CHANGELOG.md

```diff
@@ -163,6 +162,8 @@ Adding a new version? You'll need three changes:
   the controller resides in a separate Deployment. This allows the controller
   to manage its own mTLS negotiation.
   [#4942](https://github.com/Kong/kubernetes-ingress-controller/pull/4942)
+- Remove `Gateway` feature flag for Gateway API.
```

Comment:
I think more explanation to tell that `Gateway` will always on is needed here.

Reply:
Added more description to this entry. PTAL